Makefile

```diff
@@ -174,13 +174,13 @@ CONFIG_DIR := $(dir $(C))
 # EPLAT_DIR (list of external platform libraries)
 # Retrieved from P variable from the command line (paths separated by colon)
 ifeq ("$(origin P)", "command line")
-$(foreach E,$(subst :, ,$(P)), \
-$(if $(filter /%,$(E)),,$(error Path to external platform "$(E)" (P) is not absolute));\
-$(if $(wildcard $(E)), \
-	$(eval EPLAT_DIR += $(E)) \
-, $(if $(wildcard $(CONFIG_UK_BASE)/$(E)),\
-	$(eval EPLAT_DIR += $(CONFIG_UK_BASE)/$(E)), \
-	$(error Cannot find platform library: $(E)) \
+$(foreach ITR,$(subst :, ,$(P)), \
```

Comment:
Yes, `ITR` stands for iteration. I am also following the variable declaration style in our build system with capital letters. In fact, I could have used `I` for it, or `DIR`... but as you said, this comes with some risk that we are overwriting something else. So, `ITR` was a bit of a pragmatic choice since it is a bit more descriptive what it is and I have the hope that this reduces the risk of potential clashes, more than a single letter variable.
In fact, the original problem are the single letter parameters that we have. I agree that we should solve this with a different PR. Originally this comes from the `V` parameter that we use to set the verbosity. I chose `E` for `exclude`. In an earlier version I used `H` for `hide`, but I was concerned that `H` is more commonly understood  for `help`. Overall, we would have then the following single-letter build system parameters:
 `V` verbosity
 `L` library
 `P` platform
 `A` application
 `E` exclude

Reply:
Okay, then maybe let's add the vars topic into the backlog of the build system 📝

Reply:
:+1: